scripts/produceData_multiconfiguration.py

```diff
@@ -35,23 +35,44 @@ def run_cmsDriver(configdata, release):
     # else --customise {customiseUser}
     customise = f'{"" if customiseUser=="empty" else f"--customise {customiseUser}"}'
     
+    script_file = f"hgcal_tpg_validation_{configName}_{release}"
+    print("====> FILE ", script_file)
+    
     INTERVAL=int(10)
     RSS_limit=int(10000000)
     print("INTERVAL=", INTERVAL)
     print("RSS_limit=", RSS_limit)
-    command = f"echo $PWD; source /cvmfs/cms.cern.ch/cmsset_default.sh; eval `scramv1 runtime -sh`; \
-    cmsDriver.py hgcal_tpg_validation_{configName}_{release} -n {str(nbEvents)} \
-    --mc --eventcontent FEVTDEBUG --datatier GEN-SIM-DIGI-RAW \
-    --conditions {conditions} \
-    --beamspot {beamspot} \
-    --step USER:Validation/HGCalValidation/hgcalRunEmulatorValidationTPG_cff.hgcalTPGRunEmulatorValidation \
-    --geometry {geometry} --era {era} \
-    --inputCommands {inputCommands} \
-    {procMod} \
-    --filein {filein} \
-    --no_output \
-    {customise} \
-    --customise_commands {customiseCommand} & ../../../HGCTPGValidation/scripts/get_rss_memory.sh $! {INTERVAL} {RSS_limit}"
+    if (exec_flag == 1):
```

Comment:
In order to avoid duplicating code, could define the string common to both cases only once and append additional specific strings in the `no_exec` case, or use the capability of f-strings to input specific items into the string